azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/CallComposite.java

```diff
@@ -74,6 +75,43 @@ public void launch(final Context context, final GroupCallOptions groupCallOption
         );
     }
 
+    /**
+     * Launch group call composite.
+     *
+     * <pre>
+     *
+     * final CommunicationTokenRefreshOptions communicationTokenRefreshOptions =
+     *                 new CommunicationTokenRefreshOptions&#40;tokenRefresher, true&#41;;
+     * final CommunicationTokenCredential communicationTokenCredential =
+     *                 new CommunicationTokenCredential&#40;communicationTokenRefreshOptions&#41;;
+     * final GroupCallOptions groupCallOptions =
+     *                 new GroupCallOptions&#40;context, communicationTokenCredential, groupId, displayName&#41;;
+     * final LocalParticipantConfiguration config =
+     *                 new LocalParticipantConfiguration&#40;personaData&#41;;
+     * callComposite.launch&#40;groupCallOptions, config&#41;;
+     *
+     * </pre>
+     *
+     * @param context                       The android context used to start the Composite.
+     * @param groupCallOptions              The {@link GroupCallOptions} has parameters to
+     *                                      launch group call experience.
+     * @param localParticipantConfiguration The {@link LocalParticipantConfiguration} has parameters to
+     *                                      launch group call experience.
+     */
+    public void launch(final Context context,
+                       final GroupCallOptions groupCallOptions,
+                       final LocalParticipantConfiguration localParticipantConfiguration) {
```

Comment:
To me it is misleading and confusing to have a multiple option to provide a local participant name.

Reply:
Yes we had this discussion on the iOS side as well. I think we need some way to give clarity to developers, this property is not going to be sent up to ACS, but values in the `GroupCallOptions` will be sent up to the server.